docs/commands/move.md

```diff
@@ -0,0 +1,33 @@
+# `move` command
+
+Note: The implementation for `copy` and `move` are exactly the same. The only different is that move will remove the source after a successful copy.
+
+The `move` commands works like the `mv` or `rsync` binaries. It allows moving either single entries or whole folders around. Moving across mounts is supported.
+
+If the source is a directory the source directory is re-created at the destination if no trailing slash is found. Otherwise the contained secrets are flattened (similar to what `rsync` does).
```

Comment:
"If the source is a directory, (comma) the"

This is not clear to me, what do you mean with:  "if no trailing slash is found. Otherwise the contained secrets are flattened (similar to what `rsync` does)."